packages/core/src/js/feedback/FeedbackFormScreen.types.ts

```diff
@@ -0,0 +1,36 @@
+import type { TextStyle, ViewStyle } from 'react-native';
+
+export interface FeedbackFormScreenProps {
```

Comment:
Let's slightly flatten the hierarchy, by bringing the `FormText` to the root.

And where possible let's use the same naming as the JS api.
https://github.com/getsentry/sentry-javascript/blob/dafd51054d8b2ab2030fa0b16ad0fd70493b6e08/packages/feedback/src/core/integration.ts#L68

https://docs.sentry.io/platforms/javascript/guides/react/user-feedback/configuration/

Reply:
Flattened the configuration hierarchy adopting the JS api with 0258bf2c0ac778b1178da01c7b13320a2966225a